src/cueUtilities.js

```diff
@@ -422,15 +480,19 @@ module.exports = function (params, cy, api) {
         .on('free', 'node', data.eFree)
         .on('grab', 'node', data.eGrab)
         .on('position', 'node', data.ePosition)
-        .on('tap', 'node', data.eTap)
         .on('add', 'node', data.eAdd)
         .on('select', 'node', data.eSelect)
-        .on('unselect', 'ndoe', data.eUnselect)
+        .on('unselect', 'node', data.eUnselect)
         .on('free', 'node', data.eFree)
         .on('zoom pan', data.eZoom)
         .on('drag', 'node', data.eDrag);
 
       window.addEventListener('resize', data.eWindowResize);
+      $canvas.addEventListener('mousedown', interceptCytoscapeEventsWithinCue);
+      $canvas.addEventListener('mouseup', interceptCytoscapeEventsWithinCue);
+      $canvas.addEventListener('click', cueTap);
+      $canvas.addEventListener('touchstart', stopEvent);
```

Comment:
`stopEvent` here doesn't exist anymore, I think. You use a different event handler the other two times you reference `touchstart`.